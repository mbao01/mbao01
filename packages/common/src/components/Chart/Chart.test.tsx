import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import { BarChartExample } from "./stories/examples/BarChart";

describe("Chart", () => {
  const renderChart = () => render(<BarChartExample />);

  it("renders a bar chart", () => {
    const { asFragment } = renderChart();

    expect(asFragment()).toMatchSnapshot();
  });
});
