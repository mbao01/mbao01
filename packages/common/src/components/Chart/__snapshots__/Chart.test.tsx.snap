// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Chart > renders a bar chart 1`] = `
<DocumentFragment>
  <div
    class="flex aspect-video justify-center text-xs [&_.recharts-cartesian-axis-tick_text]:fill-base-content [&_.recharts-cartesian-grid_line[stroke='#ccc']]:stroke-base-content/20 [&_.recharts-curve.recharts-tooltip-cursor]:stroke-base-content/20 [&_.recharts-dot[stroke='#fff']]:stroke-transparent [&_.recharts-layer]:outline-hidden [&_.recharts-polar-grid_[stroke='#ccc']]:stroke-base-content/20 [&_.recharts-radial-bar-background-sector]:fill-base-200 [&_.recharts-rectangle.recharts-tooltip-cursor]:fill-base-200 [&_.recharts-reference-line_[stroke='#ccc']]:stroke-base-content/20 [&_.recharts-sector[stroke='#fff']]:stroke-transparent [&_.recharts-sector]:outline-hidden [&_.recharts-surface]:outline-hidden h-[250px] w-full"
    data-chart="chart-«r0»"
  >
    <style>
      
 [data-chart=chart-«r0»] {
  --color-desktop: #2563eb;
}


.dark [data-chart=chart-«r0»] {
  --color-desktop: #2563eb;
}

    </style>
    <div
      class="recharts-responsive-container"
      style="width: 100%; height: 100%; min-width: 0;"
    />
  </div>
</DocumentFragment>
`;
